ra2ce/analysis/adaptation/adaptation_option_analysis.py

```diff
@@ -63,11 +63,12 @@ def get_analysis_info(
             tuple[type[Damages | LossesBase], str]: The analysis class and the regex to find the result column.
         """
         if analysis_type == AnalysisDamagesEnum.DAMAGES:
-            return (Damages, "dam_.*")
+            # Columnname should start with "dam_" and should not end with "_segments"
```

Comment:
Having a look at the tests in `test_adaptation_option_analysis.py` it is unclear to me how this is currently being tests.

Regular expressions are cool, but truth be said not everyone understand them (quite confusing for me at times as well), ensure that this is correctly and **explicitly** tested to avoid future maintenance problems please.

Reply:
I added dedicated tests for this, matching valid column names with the regex.